encoding/ssz/query/query_test.go

```diff
@@ -0,0 +1,84 @@
+package query_test
+
+import (
+	"testing"
+
+	"github.com/OffchainLabs/prysm/v6/encoding/ssz/query"
+	"github.com/OffchainLabs/prysm/v6/encoding/ssz/query/testutil"
+	ethpb "github.com/OffchainLabs/prysm/v6/proto/prysm/v1alpha1"
+	"github.com/OffchainLabs/prysm/v6/testing/assert"
+)
+
+func TestCalculateOffsetAndLength(t *testing.T) {
+	path, err := query.ParsePath(".data.target.root")
+	assert.NoError(t, err)
```

Comment:
It is better to use `require.NoError` because with `require` the test will stop here and it doesn't make sense to continue when the call failed. Same in all other places